### PR TITLE
Fix crash when stackframe method is null

### PR DESCRIPTION
**Pull request by yousif-bugsnag**
## Goal

Fixes a crash in the notifier when a Stackframe reports a `null` method.

## Design

It's possible that `StackFrame.GetMethod()` may return `null` in some circumstances, so we should guard against this in our code

## Changeset

Added null checks wherever code accesses `StackTraceLine.Methodname`

## Testing

Tested manually by manipulating stacktraces in the debugger to cause a crash.

### CODE REVIEW

File: src/Bugsnag/Middleware.cs [hunk 1]
```diff
@@ -78,14 +78,17 @@ public static class InternalMiddleware
         {
           foreach (var stackTraceLine in exception.StackTrace)
           {
-            foreach (var @namespace in report.Configuration.ProjectNamespaces)
+            if (!Polyfills.String.IsNullOrWhiteSpace(stackTraceLine.MethodName))
```
Comment (twometresteve) [inline src/Bugsnag/Middleware.cs:81]: I think it would be worth adding another unit test case to cover the null case, given that we have a unit test framework in place for the class.

**Review by twometresteve — APPROVED** [review]
LGTM
